### DISCUSSION TITLE
[request] add support for Forwarded headers (RFC 7239)

### DISCUSSION THREAD

**Discussion by ahmadnassri** [Kong/kong#7194] [Ideas and feature requests]
[RFC 7239: Forwarded HTTP extension](http://tools.ietf.org/html/rfc7239) introduces a new Forwarded header for proxying.

this also means deprecating the `X-` prefix for those headers.

**Comment by lodrantl** [comment]
Has this been implemented anywhere yet? Or is there a way to add the Forwarded header manually.

**Comment by bungle** [comment]
@lodrantl it has not been implemented. Main reason is that Nginx real_ip module does not support it, and we currently use that. This could be added with a plugin though.
